nav2_costmap_2d/include/nav2_costmap_2d/voxel_layer.hpp

```diff
@@ -38,6 +38,9 @@
 #ifndef NAV2_COSTMAP_2D__VOXEL_LAYER_HPP_
 #define NAV2_COSTMAP_2D__VOXEL_LAYER_HPP_
 
+#include <message_filters/subscriber.h>
```

Comment:
Vector should be first

Reply:
Hmm... Linters complain if I have vector up front.